Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN mkdir /aosp/bin && \
 RUN curl https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip > /aosp/sdk-tools-linux.zip && \
     unzip /aosp/sdk-tools-linux.zip -d /aosp/sdk && \
     yes | /aosp/sdk/tools/bin/sdkmanager --licenses && \
-    /aosp/sdk/tools/bin/sdkmanager "platform-tools" "ndk;20.1.5948944" "platforms;android-32" "build-tools;30.0.0" && \
+    /aosp/sdk/tools/bin/sdkmanager "platform-tools" "ndk;20.1.5948944" "platforms;android-32" "build-tools;30.0.3" && \
```

Comment:
We recently found that google is using these components versions: https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:bazel/sdk/prebuilts.studio.sdk.BUILD;l=30?q=prebuilts.studio.sdk&ss=android-studio

It is not a suggestion to rework this line right now (however if you do change, it would be great). Just a note to not forget